inductiva/molecules/scenarios/md_water_box/post_processing.py

```diff
@@ -0,0 +1,39 @@
+"Postprocessing steps for the MDWaterBox scenario."
+import os
+import MDAnalysis as mda
+import nglview as nv
+from pathlib import Path
+
+
+class MDWaterBoxSimulationOutput:
+    """Post process the simulation output of a MDWaterBox scenario."""
+
+    def __init__(self, sim_output_path: Path = None):
```

Comment:
Is there a way to run them in VSC? Can we way to save the vizs?

Reply:
I am working on the  saving of the visualizations in another PR. The lib we are using NGLview works only on notebooks because it creates an interactive ipywidget. I am working on saving the viz using NGLview but it is creating some problems. If I can't do it with this I will do it with VMD.